options/locale/locale_en-US.ini

```
@@ -80,6 +80,7 @@ milestones = Milestones
 ok = OK
 cancel = Cancel
 rerun = Re-run
+rerun_all = Re-run All Jobs
```

```suggestion
rerun_all = Re-run all jobs
```

I  think we concluded to not fully capitalize in such sentences, but I see existing items like `Add All` violate this principle, so should be fixed later.